Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - FoodPizzaArnold
+        - PinpointerUniversal
+        - Claymore
+      rareChance: 0.3
+
+- type: entity
+  id: RandomRarelegalLoot
+  name: Random Rare Legal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetUSA
+    - type: RandomSpawner
+      prototypes:
+        - ClothingBeltSheathFilled
+        - SpaceCash10000
+        - WeaponAntiqueLaser
+        - WeaponForceGun
+        - PenCentcom
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClothingOuterCoatSpaceAsshole
+        - UnholyHalberd
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - ClothingBackpackHolding
+        - ClothingShoesBootsMagAdv
+        - OreBagOfHolding
+        - ClothingShoesBootsSpeed
+        - ClothingBackpackDuffelHolding
+        - ClothingBackpackMessengerHolding
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - AdvMopItem
+        - HandheldCrewMonitor
+        - WeaponAdvancedLaser
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
+        - PowerCellMicroreactor
+        - MuteToxinChemistryBottle
+        - SmokeGrenade
+        - ClothingShoesBootsMagBlinding
+        - TechnologyDiskRare
+        - Truncheon
+        - TurboItemRechargerCircuitboard
+        - ResearchDisk10000
+        - MedicatedSuture
+        - RegenerativeMesh
+        - FoodPizzaArnold
+        - PinpointerUniversal
+      chance: 1
+
+- type: entity
+  id: RandomEventIllegalLoot 
+  name: Random Illegal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetsyndie
+    - type: RandomSpawner
+      prototypes:
+      - DnaScramblerImplanter
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
+      - ClothingEyesGlassesOutlawGlasses
+      - SyndicatePersonalAI
+      - EnergySword
+      - WeaponPistolCobra
+      - EmpGrenade
+      - Stimpack
+      - ClusterGrenade
+      - EncryptionKeyBinary
+      - EnergyShield
+      - SyndieMiniBomb
+      - PenExplodingBox
+      - GrenadeShrapnel
+      - GrenadeIncendiary
+      - ExGrenade
+      - C4
+      - SmokeGrenade
+      - GrenadeFlashBang
+      - ClothingShoesBootsMagSyndie
+      - ClothingBackpackDuffelSyndicateEVABundle
+      - ClothingUniformJumpskirtOperative
+      - ClothingOuterVestWeb
+      - ClothingNeckScarfStripedSyndieGreen
+      - ClothingUniformJumpsuitOperative
+      - ClothingHandsGlovesBoxingRigged
+      - ScramImplanter
+      - StorageImplanter
+      - ClothingHeadHatOutlawHat
+      - ClothingBackpackDuffelSyndicateCostumeClown
```

Comment:
While it says syndicate costume clown in the item ID, it has no contraband inside it as far as I can tell, this can be a common legal loot item.  All it contains is mundane clown gear...not even an agent ID.